Exp-1/subject_results/group.retest/subj.sub_03/output.proc.sub_03

```diff
@@ -0,0 +1,936 @@
+++ 3dDeconvolve: AFNI version=AFNI_17.2.05 (Jul 27 2017) [64-bit]
+++ Authored by: B. Douglas Ward, et al.
+++ loading dataset pb04.sub_03.r01.scale+tlrc.HEAD
+++ STAT automask has 100184 voxels (out of 109350 = 91.6%)
+++ Skipping check for initial transients
+++ Input polort=4; Longest run=595.0 s; Recommended minimum polort=4 ++ OK ++
+++ -stim_times using TR=2.5 s for stimulus timing conversion
+++ -stim_times using TR=2.5 s for any -iresp output datasets
+++  [you can alter the -iresp TR via the -TR_times option]
+++ ** -stim_times NOTE ** guessing GLOBAL times if 1 time per line; LOCAL otherwise
+++ ** GUESSED ** -stim_times 1 using LOCAL times
+++ ** GUESSED ** -stim_times 2 using LOCAL times
+++ ** GUESSED ** -stim_times 3 using LOCAL times
+++ ** GUESSED ** -stim_times 4 using LOCAL times
+++ ** GUESSED ** -stim_times 5 using LOCAL times
+[7m*+ WARNING:[0m -gltsym: Can't interpret '*' scale factor in '-*Response_Control' -- replaced by 1
+[7m*+ WARNING:[0m -gltsym: '*' scale factor in '-*Response_Control' not at start of string?
+------------------------------------------------------------
+GLT matrix from 'SYM: 0.5*Correct_Task +0.5*Incorrect_Task -*Response_Control':
+          0          0          0          0          0        0.5        0.5          0          0          1          0          0          0          0          0          0          0          0          0          0          0          0 
```

Comment:
Looks you have a typo here; this should be `-Response_Control` not `-*Response_Control` in the GLT. Here you see that the contrast vector is +1 instead of -1. This probably explains why the Dice coefficient is so much larger than expected (since you are looking at scaled activation against the mean, including baseline, instead of the desired contrast).